.github/workflows/storage-layout.yaml

```
@@ -0,0 +1,38 @@
+name: "Storage Layout"
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "develop"
+
+jobs:
+  check_storage_layout:
+    name: Check storage layout
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        contract:
+          - contracts/legacy/ReserveSpenderMultiSig.sol:ReserveSpenderMultiSig
+          - contracts/legacy/StableToken.sol:StableToken
+          - contracts/legacy/Exchange.sol:Exchange
+          - contracts/legacy/GrandaMento.sol:GrandaMento
+          - contracts/swap/Broker.sol:Broker
+          - contracts/swap/BiPoolManager.sol:BiPoolManager
+          - contracts/swap/Reserve.sol:Reserve
+          - contracts/oracles/BreakerBox.sol:BreakerBox
+          - contracts/oracles/SortedOracles.sol:SortedOracles
+          - contracts/tokens/StableTokenV2.sol:StableTokenV2
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: "nightly-d369d2486f85576eec4ca41d277391dfdae21ba7"
```

copy pasta from above 

```suggestion
          version: "nightly-fd87629fbc4ae2e0fa00ccf42b4a9ebe1b521d55"
```